.github/workflows/labels.yml

```
@@ -20,7 +20,7 @@ jobs:
       GLOBAL: https://raw.githubusercontent.com/conda/infra/main/.github/global.yml
       LOCAL: .github/labels.yml
     steps:
-      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633
+      - uses: actions/checkout@f43a0e5ff2bd294095638e18286ca9a3d1956744
```

this is synced from conda/infrastructure, don't modify here

```suggestion
      - uses: actions/checkout@9bb56186c3b09b4f86b1c65136769dd318469633
```